docs/source/extensions/strategy.rst

```diff
@@ -0,0 +1,111 @@
+.. _strategy:
+
+########
+Strategy
+########
+
+Strategy depicts the training strategy to be used by the :doc:`Lightning Trainer <../common/trainer>`. It can be controlled by passing different
```

Comment:
Furthermore, we renamed them strategy as they were not related to training only. This invalidates this change ahah

Reply:
Resolved!